### DISCUSSION TITLE
wildrift-gold-efficiency/5.1/

### DISCUSSION THREAD

**Discussion by giscus** [changchiyou/wildrift-gold-efficiency#4] [Announcements]
# wildrift-gold-efficiency/5.1/

A hotfix for KALISTA, LIGHT - HORIZON FOCUS, LIGHT - INFINITY ORB

https://changchiyou.github.io/wildrift-gold-efficiency/5.1/

**Comment by changchiyou** [comment]
[WILD RIFT PATCH NOTES 5.1 - ITEM CHANGES & THEME: LIGHT AND RUIN](https://wildrift.leagueoflegends.com/en-us/news/game-updates/wild-rift-patch-notes-5-1/#item-changes)

---

#### NEEDLESSLY LARGE ROD

_Base Stats_

-   Ability Power: 65 → **60**

#### BLASTING WAND

_Base Stats_

-   Ability Power: 45 → **40**

---

#### LIGHT - HORIZON FOCUS

_Base Stats_

-   Cost: 2800g → **3100g**
-   Ability Power: 80 → **90**
-   Maximum Health: 150 → **0**
-   Ability Haste: 15 → **20**

_Build Path_

-   **\[New\]** Ring of Revelation (400g) + Blasting Wand (900g) + Fiendish Codex (1000g) + 800g

_Hypershot_

-   **\[Removed\]** Apply 1 mark to an enemy champion when you damage them with a non-targeted ability from 500 units away. Apply 2 marks for immobilizing them, regardless of distance. Marked enemies are revealed and detonated at 3 stacks, inflicting a new mark and dealing 90 + 25% Ability Power as magic damage to the target.
-   **\[New\]** Damaging an enemy champion with an ability from 600 units away reveals them for 6 seconds and increases damage dealt to them by 12%.
-   **\[New\]** Focus: When Hypershot is triggered, it reveals all enemy champions within 1,200 units of the target for 2s. (15s cooldown)

#### LIGHT - INFINITY ORB

_Base Stats_

-   Cost: 2900g → **3200g**
-   Ability Power: 85 → **95**

_Build Path_

-   **\[New\]** Aether Wisp (950g) + Prophet's Pendant (1000g) + Amplifying Tome (500g) + 750g

_Inevitable Demise_

-   Abilities and empowered attacks Critically Strike for 20% bonus damage against enemies below 35% → **45% Health**

#### LIGHT - HARMONIC ECHO

_Base Stats_

-   Cost: 2700g → **3000g**

_Harmonic Echo_

-   Moving and casting abilities builds Harmony. At 100 Harmony, your next healing or shielding ability cast on an ally restores 130 → **135** + 10% Ability Power as Health to your target and up to 3 nearby allied champions.
-   **\[New\]** Every 10 Ability Haste you have increases healing by 1%.

#### LIGHT - RIFTMAKER

_Base Stats_

-   Cost: 3200g → **3300g**
-   Ability Power: 80 → **90**

_Build Path_

-   **\[New\]** Fiendish Codex (100g) + Hextech Alternator (1400g) + Amplifying Tome (500g) + 400g

_Void Corruption_

-   When in combat with champions, gain 1 stack of Corruption every second. Each stack increases the damage you deal by 3% → **2.5%**, up to 3 → **4** stacks. At maximum stacks, the additional damage provided by Corruption becomes true damage

#### LIGHT - YOUMUU’S GHOSTBLADE

_Base Stats_

-   Cost: 3000g → **3200g**

_Momentum_

-   Moving builds Momentum, granting up to 40 → **50** Movement Speed and **5-10 Armor Pen** at 100 stacks. Attacking removes all Momentum, **but Armor Pen will last for 3 seconds after**.

#### LIGHT - DAWNSHROUD

_Base Stats_

-   Cost: 2700g → **3000g**

_Dawnbringer_

-   If you are within 400 units of an enemy champion, and you immobilize a champion or are immobilized, reveal all nearby enemy champions for 3 seconds and deal 80 + 5% bonus Health as magic damage, **and heal yourself for the same amount for 3s**. (3s cooldown)

#### RUIN - RABADON’S DEATHCAP

_Base Stats_

-   Ability Power: 100 → **90**

_Overkill_

-   Increases Ability Power by 20 - 45% → **20 - 55%** (based on level)

**_\[New\]_** _Cinders_

-   Reduces your base Health by 10 - 100.

#### RUIN - BLADE OF THE RUINED KING

_Base Stats_

-   Cost: 3200g → **3000g**

_Thirst_

-   +10% → **12%** Physical Vamp

_Ruined Strikes_

-   Attacks deal bonus physical damage equal to 7% of the enemy's current Health on-hit. (Melee attacks deal 10% → **9%**). Minimum damage: 15. Maximum damage vs. monsters: 60.

_Drain_

-   Hitting a champion with 3 attacks or abilities deals 30-100 bonus magic damage and steals 25% → **40%** of their Movement Speed for 2s. (60s cooldown)

**_\[New\]_** _Repentance_

-   Reduces your base Health by 10 - 100.

#### RUIN - INFINITY EDGE

_Base Stats_

-   Cost: 3400g → **3200g**

_Infinity_

-   Critical Strikes deal 205% → **225%** damage instead of 175%.

**_\[New\]_** _Self-Destruction_

-   Reduces your base Health by 10 - 100.

#### RUIN - AMARANTH’S TWINGUARD

_Endurance_

-   Gain 1 stack of Endurance every second while in combat with enemy champions (max 5 → **3** stacks). At maximum stacks, gain 20% size, 20% Tenacity, and increase Armor by 30% → **35%** and Magic Resist by 30% → **35%** until out of combat with champions

**_\[New\]_** _Withering_

-   Reduces your Attack Damage by 1 - 15

#### RUIN - WARMOG’S ARMOR

_Warmog's Heart_

-   If you have at least 950 → **1150** bonus Health and have not taken any damage during the last 6 → **4**, restore 5% maximum Health per second.

**_\[New\]_** _Strength of Will_

-   Restores 0.3% max Health per second.

**_\[New\]_** _Dead End_

-   Reduces your Attack Damage by 1 - 15.

#### RUIN - STERAK’S GAGE

_Sterak's Fury_

-   Triggering Lifeline increases size, **empowers you, removes all crowd control effects on you (except Airborne),** and grants 30% → **50%** Tenacity for 8 → **4** seconds

_**\[New\]** Guilt Bearer_

-   Reduces your Movement Speed by 5

**Comment by changchiyou** [comment]
[WILD RIFT PATCH NOTES 5.1 - HOTFIX](https://wildrift.leagueoflegends.com/en-us/news/game-updates/wild-rift-patch-notes-5-1/#hotfix)

---

#### LIGHT - HORIZON FOCUS

Horizon Focus has been doing too much damage so we are adjusting its numbers slightly.

_Hypershot_

-   Damaging an enemy champion with an ability from 600 units away reveals them for 6 seconds and increases damage dealt to them by 12% → **9%**

#### LIGHT - INFINITY ORB

Infinity Orb’s price isn’t accurately reflecting the amount of value offered so we’ve adjusted it as well.

_Base Stats_

-   Cost: 3200 → **3400g**
